### ISSUE TITLE
Code-related

### ISSUE THREAD

**Issue by aleckgene** [SaranyaGopinath178/Offline-Signature-Verification-Artificial-Neural-Network#1]
A code is not used in the Integrated Code namely function_triSurface_six_fold.m. I also read your paper, there's a missing feature there namely 'Transition Feature.' I am also developing a signature verification as a research. Can you help me with this? Thanks.

**Comment by ChandanaPulusu** [comment]
Yes! could you please provide the missing feature there namely 'Transition Feature.' Target and input data to nn also please!!!
